insert_urls.py

```diff
@@ -6,14 +6,12 @@
 
 
 def create_connection(db_file):
-    conn = None
     try:
-        conn = sqlite3.connect(db_file)
-        return conn
+        return sqlite3.connect(db_file)
     except Error as e:
         print(e)
 
-    return conn
+    return None
```

Comment:
By default, it returns a `None` value and there is no need for `None`.

Reply:
no no :)
By default, None is not be returned
If it goes to the except block, it only prints the error
But in the main function, we consider this function as the value of the con variable.

On the other hand, if you see the PR code, I deleted the conn variable from inside the create_connection function
But in the repo code, the con variable is returned, which is the default value of None.